notebooks/search/08-learning-to-rank.ipynb

```diff
@@ -0,0 +1,1527 @@
+       " ('Star Wars: The Last Jedi', 8.053349, '181808'),\n",
+       " ('Solo: A Star Wars Story', 8.053349, '348350'),\n",
+       " ('The Star Wars Holiday Special', 8.053349, '74849'),\n",
+       " ('Phineas and Ferb: Star Wars', 8.053349, '392216')]"
+      ]
+     },
+     "execution_count": 40,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "query = \"star wars\"\n",
+    "\n",
+    "# First let's display the result when not using the rescorer:\n",
+    "search_fields = [\"title\", \"overview\", \"actors\", \"director\", \"tags\", \"characters\"]\n",
+    "bm25_query = { \"multi_match\": { \"query\": query, \"fields\": search_fields } }\n",
+    "\n",
+    "bm25_search_response = es_client.search(index=MOVIE_INDEX, query=bm25_query)\n",
+    "\n",
+    "[\n",
+    "    (movie[\"_source\"][\"title\"], movie[\"_score\"], movie[\"_id\"])\n",
+    "    for movie in bm25_search_response['hits']['hits']\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[('Star Wars', 4.1874104, '11'),\n",
+       " ('Star Wars: The Clone Wars', 2.3627238, '12180'),\n",
+       " ('Star Wars: The Rise of Skywalker', 1.7667875, '181812'),\n",
+       " ('Star Wars: The Force Awakens', 1.3336482, '140607'),\n",
+       " ('Star Wars: The Last Jedi', 1.3336482, '181808'),\n",
+       " ('Rogue One: A Star Wars Story', 1.1134433, '330459'),\n",
+       " ('LEGO Star Wars Summer Vacation', 1.082971, '980804'),\n",
+       " (\"Doraemon: Nobita's Little Star Wars 2021\", 0.9138395, '782054'),\n",
+       " ('LEGO Star Wars Terrifying Tales', 0.89640737, '857702'),\n",
+       " ('Solo: A Star Wars Story', 0.65811557, '348350')]"
+      ]
+     },
+     "execution_count": 41,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Now let's display result when using the rescorer:\n",
+    "\n",
+    "ltr_rescorer = {\n",
+    "    \"learning_to_rank\": {\n",
+    "        \"model_id\": LEARNING_TO_RANK_MODEL_ID,\n",
+    "        \"params\": {\"query\": query},\n",
+    "    },\n",
+    "    \"window_size\": 100,\n",
+    "}\n",
+    "\n",
+    "rescored_search_response = es_client.search(index=MOVIE_INDEX, query=bm25_query, rescore=ltr_rescorer)\n",
+    "\n",
+    "[\n",
+    "    (movie[\"_source\"][\"title\"], movie[\"_score\"], movie[\"_id\"])\n",
+    "    for movie in rescored_search_response[\"hits\"][\"hits\"]\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We saw above that the title and popularity fields are important ranking feature in our model. Here we can see that now all results contain the query terms in the title. Moreover, more popular movies rank higher, for example `Star Wars: Episode I - The Phantom Menace` is now in third position."
```

Comment:
@afoucret I saw that in the version that got merged `Star Wars: Episode I - The Phantom Menace` is actually not in the result list. I think it would be good to 
- update the example with a movie that is actually here
- set `random.seed` to a fixed number early in the notebook so that parts that use random values always do the same for each run.

What do you think?